SPDX-FileCopyrightText: 2022 fosslinux <fosslinux@aussies.space>

SPDX-License-Identifier: Python-2.0.1

musl (correctly) implements the POSIX posix_close function, however
this was added well after Python 2.0.1 was released.

--- Modules/posixmodule.c	2022-09-16 16:46:09.809812072 +1000
+++ Modules/posixmodule.c	2022-09-16 16:47:23.254166370 +1000
@@ -3267,12 +3267,12 @@
 }
 
 
-static char posix_close__doc__[] =
+static char py_posix_close__doc__[] =
 "close(fd) -> None\n\
 Close a file descriptor (for low level IO).";
 
 static PyObject *
-posix_close(PyObject *self, PyObject *args)
+py_posix_close(PyObject *self, PyObject *args)
 {
 	int fd, res;
 	if (!PyArg_ParseTuple(args, "i:close", &fd))
@@ -5300,7 +5300,7 @@
 	{"tcsetpgrp",	posix_tcsetpgrp, METH_VARARGS, posix_tcsetpgrp__doc__},
 #endif /* HAVE_TCSETPGRP */
 	{"open",	posix_open, METH_VARARGS, posix_open__doc__},
-	{"close",	posix_close, METH_VARARGS, posix_close__doc__},
+	{"close",	py_posix_close, METH_VARARGS, py_posix_close__doc__},
 	{"dup",		posix_dup, METH_VARARGS, posix_dup__doc__},
 	{"dup2",	posix_dup2, METH_VARARGS, posix_dup2__doc__},
 	{"lseek",	posix_lseek, METH_VARARGS, posix_lseek__doc__},
